tubearchivist/home/views.py

```diff
@@ -759,13 +760,16 @@ def _update_view_data(self, playlist_id, playlist_info):
             + "{return params.scores[doc['youtube_id'].value];} "
             + "return 100000;"
         )
+        if playlist_info["playlist_type"] == "custom":
```

Comment:
I'm not following here: Why is that addition needed here? The original query was matching against the playlist id, custom playlists should be able to do that aswell. Or am I missing something?

Reply:
you may be correct. When we change the position of a video in a custom playlist, we always re-set the idx attribute of all videos in the playlist. So the original logic in this function should be ok. I think I had a previous implementation where I was relying on the order of the videos in the stored array. Using idx to identify the order of videos is more consistent to what is already coded. So perhaps this is a leftover of the previous thinking.

Reply:
sounds good, undid the changes.